Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -346,14 +346,71 @@ modes and can be enabled as follows for ``nodePort.mode=hybrid`` in this example
         --set global.k8sServiceHost=API_SERVER_IP \\
         --set global.k8sServicePort=API_SERVER_PORT
 
+A list of drivers supporting native XDP can be found in below table. The corresponding
+network driver name of an interface can be determined as follows:
+
+.. parsed-literal::
+
+    # ethtool -i eth0
+    driver: nfp
+    [...]
+
++-------------------+------------+-------------+
+| Vendor            | Driver     | XDP Support |
++===================+============+=============+
+| Amazon            | ena        | 5.6+        |
++-------------------+------------+-------------+
+| Broadcom          | bnxt_en    | 4.11+       |
++-------------------+------------+-------------+
+| Cavium            | thunderx   | 4.12+       |
++-------------------+------------+-------------+
+| Freescale         | dpaa2      | 5.0+        |
++-------------------+------------+-------------+
+| Intel             | ixgbe      | 4.12+       |
+|                   +------------+-------------+
+|                   | ixgbevf    | 4.17+       |
+|                   +------------+-------------+
+|                   | i40e       | 4.13+       |
+|                   +------------+-------------+
+|                   | ice        | 5.5+        |
++-------------------+------------+-------------+
+| Marvell           | mvneta     | 5.5+        |
++-------------------+------------+-------------+
+| Mellanox          | mlx4       | 4.8+        |
+|                   +------------+-------------+
+|                   | mlx5       | 4.9+        |
++-------------------+------------+-------------+
+| Microsoft         | hv_netvsc  | 5.6+        |
++-------------------+------------+-------------+
+| Netronome         | nfp        | 4.10+       |
++-------------------+------------+-------------+
+| Others            | virtio_net | 4.10+       |
+|                   +------------+-------------+
+|                   | tun/tap    | 4.14+       |
++-------------------+------------+-------------+
+| Qlogic            | qede       | 4.10+       |
++-------------------+------------+-------------+
+| Socionext         | netsec     | 5.3+        |
++-------------------+------------+-------------+
+| Solarflare        | sfc        | 5.5+        |
++-------------------+------------+-------------+
+| Texas Instruments | cpsw       | 5.3+        |
```

Comment:
nit: In other parts of the docs, we represent versions that mean "5.3 or later" as `>= 5.3`

Reply:
Thanks for the review, ok, I'll change it into that.